fix(faq): correct glacier mentions MTA-1974 (#653)
scaleway/docs-content: faq/coldstorage.mdx

---
meta:
 title: Cold Storage
 description: Discover Scaleway Glacier and its features.
content:
 h1: Cold Storage
 paragraph: Discover Scaleway Glacier and its features.
image: coldstorage.webp
---

## What is the cold storage Glacier storage class?

The cold storage “Glacier” class of Object Storage gets its name in opposition to “hot data”. Hot data is the type that is frequently accessed. Cold storage is accessed less frequently and does not require fast access like hot data. 

Scaleway’s cold storage service Glacier is engineered on specific hardware:

- SMR disks that increase storage density and overall per-drive storage capacity

> - motherboards manufactured internally that can power on/off disks on-demand with a SATA bus tree matrix.

Reviewer: ```suggestion
- motherboards manufactured internally that can power disks on/off on-demand with a SATA bus tree matrix.
```

> This particular hardware configuration allows us to offer competitive pricing for use-cases that do not need the performance of other Object storage classes. The counterpart is that data stored on a cold storage service such as Scaleway Glacier has [limitations]() in relation to data retrieval.

Reviewer 2: There's a missing link here for the limitations.

## When should I use Glacier class instead of Standard class?
 

> Scaleway Glacier is engineered for use-cases that can be described as “deep archiving”: a durable Object Storage class designed for data archiving, legal documents of long-term backups that do not require immediate access.

Reviewer: ```suggestion
Scaleway Glacier is engineered for use-cases that can be described as “deep archiving”. It's a durable Object Storage class designed for data archiving, for example legal documents or long-term backups that do not require immediate access.
```

Reviewer: meh reading this back I don't like my suggestion either

Standard class is ideal to store data that is frequently accessed, on-demand content such as streaming or Content Delivery Networks (CDN).
 
## I have started the restoration of an object, why is it not available instantaneously?
 
An object stored in Glacier class is listed but cannot be downloaded instantly. It needs to be restored to Standard class first.
We recommend that you use Glacier to archive data that you may need only once or twice within a decade. 

<Message type="important">

  It can take anywhere from a few seconds to 6 hours to retrieve the first byte of a requested file. From that point, file retrieval is limited to xMo/sec. Unfortunately, we are unable to provide estimates of how long data retrieval will therefore take for each use case.

</Message>

 
## Is there a free tier on Scaleway Glacier?
 

> Yes, there is a free tier and you can benefit from 75 GB of storage in Glacier class for free with no time limit. In addition, you can also benefit from  75 GB of storage and 75 GB of outgoing and inter-regional transfer in Object Storage’s standard class.

Reviewer: ```suggestion
Yes, there is a free tier and you can benefit from 75 GB of storage in Glacier class for free with no time limit. In addition, you can also benefit from 75 GB of storage and 75 GB of outgoing and inter-regional transfer in Object Storage’s standard class.
```

 
## Can I upload an object directly to Glacier?
 
Yes, you can select the storage class (`STANDARD` or `GLACIER`) when uploading objects using the Scaleway console. It is also possible to do so using the API. For more information, refer to our [feature documentation](/storage/object/how-to/edit-storage-class/).
 
## Why are Glacier objects stored in Paris DC4? Is it accessible from buckets in all locations?
 
Our Paris underground databunker (DC4) provides an increased layer of security by being 25 meters deep below the surface in a bunker where the Glacier storage class cluster is available.
This storage cluster is available from any bucket in Paris and Amsterdam and all objects can change the storage class to Glacier.
 
<Message type="important">
 
It is not yet possible to transfer files to Glacier in Warsaw, Poland.
 
</Message>

 
## When are lifecycle management rules applied?
 
The expiration & transition rules are implemented every day at midnight UTC, but the actual action might take between one minute and twenty-four hours* after implementation.
 
For example, a user wants to configure a lifecycle rule that moves an object after two days to Glacier:
 

> - lifecycle Rules are checked everyday at midnight.

Reviewer: is it right that lifecycle isn't capitalized and rules is?

- the user uploaded a file on April 6th, at 2:00 pm UTC.
- the rule is triggered on April 8th, at 11:59 pm UTC.
- the application of the rule can take up to 24 hours
⇒ the object will be transferred to Glacier no later than 11:59 pm UTC on April 9th.
 
<Message type="note">
 
When a lifecycle rule is set for a very large amount of objects (several 100000s) simultaneously, the rule might take longer to be applied.
 
</Message>

 
## How do I encrypt my data?
 

> For increased security, we recommend encrypt your data before sending it to Scaleway Glacier.

Reviewer 2: ```suggestion
For increased security, we recommend you encrypt your data before sending it to Scaleway Glacier.
```

Encryption should be done on the client-side to guarantee that there is no evidence of your private key stored on the cluster.
Data can be encrypted using third-party solutions, like GPG. It is possible to automatize your backups and to encrypt them using a solution like Duplicity or Rclone using the crypt backend.
 
## Can I manage my storage and archiving constraints related to the GDPR with Scaleway Glacier?
 
The GDPR requires that the life span of data be defined in two distinct cases:

- the deletion of data after a certain period of time
- the archiving of data that must legally be kept

> Lifecycle management features meet both of these requirements by allowing you to set a timeframe beyond which your data are archived or deleted.

Reviewer: ```suggestion
Lifecycle management features meet both of these requirements by allowing you to set a timeframe beyond which your data is archived or deleted.
```

 
## Why is Scaleway Glacier technology much more reliable than tape storage technologies?
 
Access to tape storage platforms requires a tape library to identify the cartridge on which the data is stored. Furthermore, retrieving data stored using tape storage requires reading several kilometers of tape.
 
Scaleway Glacier offers extra benefits compared to tape storage:

- Scaleway Glacier’s hard disk drive-based technology offers much better random data access time
- Scaleway Glacier enables parallel writing to multiple disks simultaneously, providing very high read and write speeds. 
- redundancy is achieved by advanced parity mechanisms (6+3) on many media, whereas tape technology does not allow this and is limited to CRC. 
- there is no risk of mechanical part failure (ex: robot) or wear and tear (tape drive).

## Can I change the storage class of an entire bucket from Standard to Glacier?
 
Yes, a best practice is to create a transition lifecycle rule with an empty prefix.
In this case, all files contained within the selected bucket will have their storage class altered automatically on the dates stipulated by you.

> However, due to S3 Protocol restrictions, a lifecycle rule cannot be created to modify the storage class from glacier to standard.

Reviewer 2: ```suggestion
However, due to S3 Protocol restrictions, a lifecycle rule cannot be created to modify the storage class from Glacier to Standard.
```
